### ISSUE TITLE
MacOS 10.14.5 doesn't include xCode CLT by default

### ISSUE THREAD

**Issue by Jarli01** [Homebrew/brew#6215]
**Please note that we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [X] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [N/A] ran a `brew` command and reproduced the problem with multiple formulae? If it's a problem with a single, official formula (not cask) please file this issue at Homebrew/homebrew-core: https://github.com/Homebrew/homebrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [N/A] ran `brew update` and can still reproduce the problem?
- [N/A] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [N/A] ran `brew config` and `brew doctor` and included their output with your issue?

Use the single line installation on a brand new install (same occurs from shrink wrapped) installations with MacOS 10.14.5, results in "Can not install brew because it is not currently available from the software update server"

# What you were trying to do (and why)

Install brew with ```/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"``` even with ```xcode-select --install``` run first and than installing brew fails. 

# What happened (include command output)

Can’t install the software because it is not currently available from the Software Update server.

TL:DR you have to install Xcode CLT manually (I've already scripted this, but multiple topics have been opened on brew.sh regarding this)

# What you expected to happen

The single line installation process to work. 

# Step-by-step reproduction instructions (by running `brew` commands)

```/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"``` on a brand new, unmolested installation of Mac OSX 10.14.5

**Comment by NemesisOfNo** [comment]
xcode-select —install fails for me. This started on Tuesday. I noticed that there was a beta for the CLT on Monday during wwdc, but then it disappeared. 

I confirmed on a network with caching server I cannot download the pkg and confirmed from home I cannot either. 

I am willing to bet the update catalog from Apple is pointing to the beta that was pulled.

**Comment by MikeMcQuaid** [comment]
We cannot fix this. Only Apple can.

**Comment by Jarli01** [comment]
What I've done is logged into the developer portal and manually downloaded the dependency and scripted that install before brew attempts its install. 

Not ideal, I know, but here it is for anyone who needs it. 

```hdiutil attach /Path to Command_Line_Tools_macOS_10.14_for_Xcode_10.2.dmg```

```sudo installer -pkg /Volumes/Command\ Line\ Developer\ Tools/Command\ Line\ Tools\ \{macOS\ Mojave\ version\ 10.14\).pkg -target "/"```

```hdiutil detach /Volume/Command*```

```/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"````

```remainder of script```

**Comment by Jarli01** [comment]
> Does someone have the ability to open a Radar issue with Apple or talk to someone at WWDC about this?

I have a developer account, can you provide some insight into what specifically is failing as I don't have anything to point the apple team too.

**Comment by mistydemeo** [comment]
I submitted a feedback report - `rdar://6128607`

**Comment by Jarli01** [comment]
@mistydemeo any update from Apple on this?

**Comment by dive** [comment]
This is the [direct link](https://download.developer.apple.com/Developer_Tools/Command_Line_Tools_for_Xcode_11_Beta/Command_Line_Tools_for_Xcode_11_Beta.dmg) to download Command Line Tools for Xcode 11 Beta. Works more or less fine to me on macOS 10.15 beta.
I do not recommend to install Command Line Tools for Xcode 10.2.1 on macOS 10.15, there are some internal changes related to the migration to Python 3 and other stuff; so, in some cases, Xcode 10.2.1 tools produce some unpredictable results to me (not related to Homebrew, but still).

**Comment by MikeMcQuaid** [comment]
Fixed by Apple now.

**Comment by rdp** [comment]
Seems working here...

**Comment by Jarli01** [comment]
@rdp that's because apple FTS and this ticket has been closed for 9 days now.
